Note claude.ai web app limitation for the Qdrant Advisor (#102)
qdrant/skills: README.md

The Advisor ships no static content of its own. When you raise a Qdrant problem, it searches [skills.qdrant.tech](https://skills.qdrant.tech) live, traverses the skill hierarchy along the branch that matches your symptom, and grounds its diagnosis in the current, authoritative guidance, loading only the relevant context. Because it fetches fresh every session, you don't need to reinstall to stay current, and you don't have to remember a URL or hope the site is in the model's training data.

> > **Using the claude.ai web app?** There, the Advisor can't fetch `skills.qdrant.tech` on its own. You need to add `Use skills.qdrant.tech` to your prompt (refer to [Pass the URL directly](#alternative-pass-the-url-directly)). In Claude Code and the desktop app it works as described above.

Author: ```suggestion
> **Using the claude.ai web app?** The Advisor can't fetch `skills.qdrant.tech` on its own when you are using the web app. You need to add `Use skills.qdrant.tech` to your prompt (refer to [Pass the URL directly](#alternative-pass-the-url-directly)).
```
